chart/env/prod.yaml

```diff
@@ -101,6 +106,16 @@ mongodbMigration:
     limits:
       cpu: 1
 
+# --- jobs (post-upgrade hooks) ---
+
+cacheMaintenance:
+  # action: "backfill" -- Set this value according to required action for maintenance
```

Comment:
> Set this value according to required action for maintenance

where?

Shouldn't it be set here, and if we need another action to be run, change it here too?

Reply:
I fixed it on https://github.com/huggingface/datasets-server/pull/1017/commits/e67005f028c2117e44c80a88f7d170ab664afaef